docs/topics/modules/planning_development_work.adoc

```diff
@@ -0,0 +1,91 @@
+[id="planning_development_work"]
+= Planning your development work
+
+After creating a space, use the {osio} planner to plan your development work and to ensure efficient execution of your project.
+
+NOTE: The development process you select, while creating the space, determines the planner's guided work item type hierarchy.
+
+This example walks you through the Scenario Driven Development planning process to plan an end to end development flow of launching and deploying a sample application.
+
+. In the dashboard of your space, select the *Plan* tab at the top of the page to view the planner. The default *Backlog* view displays the *Scenarios* work item type group for your space.
+. Select the *Show Tree* check box to see the work items in a tree view. The tree view allows you to quickly add work items according to the guided hierarchy using the quick-add and in-line add options.
+. Use the quick-add option to create a *Scenario* as follows:
+.. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
++
+image::quickadd_wi_type.png[Quick Add work item type]
++
+.. In the *Work Item Title* field, add `Enable an end to end, quick, developer work-flow, within a continuous integration and deployment model` as the title.
+.. Press kbd:[Enter] to create the scenario.
++
+image::gs_scenario_eg.png[Example Scenario]
++
+
+. Similarly, select *Fundamental* in the *Add Work Item* pane, to create a fundamental work item with the title `The application should be secure with appropriate licenses`.
++
+image::gs_fundamental_eg.png[Example Fundamental]
++
+
+. Create child *Experience* and *Value Proposition* for the *Scenario* using the in-line add option as follows:
+.. In the work item list, click the (+) icon adjacent to the scenario to see the in-line *Add Child Work Item* pane below it.
++
+image::gs_inline-add_eg.png[Inline Add]
++
+
+.. Select *Experience* from the drop-down list, add the title `User should be able to launch any application through a simple work-flow`, and Press kbd:[Enter].
+.. Similarly, select *Value Proposition* and add the title `Easy end to end developer experience`.
++
+image::gs_exp_vp_egs.png[Example Experiences and Value Propositions]
++
+NOTE: A Scenario can have multiple realizing experiences and supporting value-propositions.
++
+
+. Create a child *Feature* to actualize the *Experience* using the in-line add option with the title `User should be able to easily launch a sample vertx application using a smooth developer work-flow`.
```

Comment:
is actualize a technical term here? It sounds like marketing speak. Plan the experience? Execute the experience? Something else?